modules/custom/openy_activity_finder/src/Plugin/search_api/processor/PartsOfDay.php

```diff
@@ -1,89 +0,0 @@
-<?php
```

Comment:
Why did you remove this file?

Reply:
Accidentally during massive merge, returned